Exiled.API/Features/DamageHandlers/AttackerDamageHandler.cs

```diff
@@ -17,7 +17,7 @@ namespace Exiled.API.Features.DamageHandlers
     public abstract class AttackerDamageHandler : DamageHandler
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="AttackerDamageHandler"/> class.
+        /// Initializes a new instance of <see cref="AttackerDamageHandler"/> class.
```

Comment:
```suggestion
        /// Initializes a new instance of the <see cref="AttackerDamageHandler"/> class.
```
"the" is appropriate here - see above comments

Reply:
Reverted